projects/cm_mcu/AlarmUtilities.c

```diff
@@ -138,4 +138,163 @@ struct GenericAlarmParams_t tempAlarmTask = {
     .checkStatus = &TempStatus,
     .errorlog_registererror = &TempErrorLog,
     .errorlog_clearerror = &TempClearErrorLog,
+    .stack_size = 176,
+};
+
+///////////////////////////////////////////////////////////
+//
+// Voltage Alarms
+//
+///////////////////////////////////////////////////////////
+
+// current value of the thresholds
+#define INITIAL_ALARM_VOLT_PERCENT 0.05f // +/-5% from the ADC thresholds
+static float alarmVolt = INITIAL_ALARM_VOLT_PERCENT;
+
+float getAlarmVoltage(void)
+{
+  return alarmVolt;
+}
+void setAlarmVoltage(float voltthres)
+{
+  alarmVolt = voltthres;
+}
+
+// current status of voltages
+static uint8_t currentVoltStatus[3] = {0.0, 0.0, 0.0}; // FIXME : bitmasks for voltage statuses
```

Comment:
Can you make these into unsigned integers not floats (i.e., 0.0 -> 0U)

Reply:
Done.